<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCMissionDoDigicamControl</class>
 <widget class="QWidget" name="QGCMissionDoDigicamControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1547</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDoubleSpinBox" name="param1SpinBox">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>0</height>
      </size>
     </property>
     <property name="prefix">
      <string extracomment="0 = off 1 = on">off/on </string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="param4SpinBox">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>0</height>
      </size>
     </property>
     <property name="prefix">
      <string extracomment="0=Relock 1=Ignore 2=Lock ">Focus Lock </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="param5SpinBox">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="prefix">
      <string extracomment="&gt;0 to trigger"> Shutter </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
